paper/src/main/java/com/denizenscript/denizen/paper/events/EntityKnocksbackEntityScriptEvent.java

```diff
@@ -33,6 +36,8 @@ public class EntityKnocksbackEntityScriptEvent extends BukkitScriptEvent impleme
     // <context.entity> returns the EntityTag that was knocked back.
     // <context.damager> returns the EntityTag of the one who knocked.
     // <context.acceleration> returns the knockback applied as a vector.
+    // <context.cause> returns the cause of the knockback (only on 1.20 and above).
+    // Causes list: <@link url https://jd.papermc.io/paper/1.21.1/io/papermc/paper/event/entity/EntityKnockbackEvent.Cause.html>

```

Comment:
The format is usually `MC 1.20+` iirc, also context meta should stay in single lines where possible